bool serial_fail
bool ahrs_fail
bool gps_fail
bool armed
float32 pitch
float32 roll
float32 yaw